ogr/services/forgejo/issue.py

```diff
@@ -37,4 +145,46 @@ def get_list(
         assignee: Optional[str] = None,
         labels: Optional[list[str]] = None,
     ) -> list["Issue"]:
-        raise NotImplementedError("TBD")
+        if not project.has_issues:
+            raise IssueTrackerDisabled()
+
+        parameters: dict[str, Union[str, list[str], bool]] = {
+            "state": status.name,
+            "type": "issues",
+        }
+        if author:
+            parameters["created_by"] = author
+        if assignee:
+            parameters["assigned_by"] = assignee
+        if labels:
+            parameters["labels"] = labels
+        try:
+            issues_paginator = paginate(
+                project.service.api.issue.list_issues,
+                owner=project.namespace,
+                repo=project.repo,
+                **parameters,
+            )
+            return [ForgejoIssue(issue, project) for issue in issues_paginator]
+        except NotFoundError as ex:
+            if "user does not exist" in str(ex):
+                return []
+            raise OperationNotSupported(f"Failed to list issues {ex}") from ex
+
+    def close(self) -> "Issue":
+        self.partial_api(self.api.edit_issue)(state="closed")
+        self.__update_info()
+        return self
+
+    def add_assignee(self, *assignees: str) -> None:
+        current_assignees = [
+            assignee.login if hasattr(assignee, "login") else assignee
+            for assignee in self.assignees
+        ]
+        updated_assignees = list(set(current_assignees + list(assignees)))
+        self.partial_api(self.api.edit_issue)(assignees=updated_assignees)
+        self.__update_info()
+
+    def add_label(self, *labels: str) -> None:
+        self.partial_api(self.api.add_label)(labels=labels, index=self._index)
```

Comment:
Why do you _hack around_ the index (issue number) above, when you pass it here anyways?